test/Libraries/RevitNodesTests/Elements/ElevationMarkerTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+using Autodesk.DesignScript.Geometry;
+using Autodesk.Revit.DB;
+
+using NUnit.Framework;
+
+using Revit.Elements;
+using Revit.GeometryReferences;
+
+using RevitServices.Persistence;
+using RevitServices.Transactions;
+using RevitTestServices;
+
+using RTF.Framework;
+
+namespace RevitNodesTests.Elements
+{
+    [TestFixture]
+    class ElevationMarkerTests : RevitNodeTestBase
+    {
+        [Test]
+        [TestModel(@".\ElevationMarker\elevationMarkerTests.rvt")]
+        public void CanCreateElevationMarker()
+        {
+            // Arrange
+            var room = ElementSelector.ByElementId(316289, true);
+            var elevationViewFamilyType = ElementSelector.ByElementId(86228, true);
+
+            var expectedType = typeof(Revit.Elements.ElevationMarker);
+            var expectedLocation = Autodesk.DesignScript.Geometry.Point.ByCoordinates(1392.616, 915.356, 0.000);
+
+            // Act
+            var elevationMarkerLocation = room.GetLocation() as Autodesk.DesignScript.Geometry.Point;
+            var elevationMarker = Revit.Elements.ElevationMarker.Create(elevationViewFamilyType, elevationMarkerLocation, 100);
+
+            var newElevationMarkerType = elevationMarker.GetType();
+
+            // Assert
+            Assert.AreEqual(expectedType, newElevationMarkerType);
+        }
+
+        [Test]
+        [TestModel(@".\ElevationMarker\elevationMarkerTests.rvt")]
+        public void CanCreateElevationViewsByMarkerIndex()
+        {
+            // Arrange
+            var elevationMarker = ElementSelector.ByElementId(327339, true) as Revit.Elements.ElevationMarker;
+            var planView = ElementSelector.ByElementId(312, true) as Revit.Elements.Views.View;
+
+            var expectedViewName = "Elevation 8 - a";
+
+            // Act
+            var elevationView = elevationMarker.CreateElevationByMarkerIndex(planView, 0);
+            var elevationViewName = elevationView.Name;
+
+            // Assert
+            Assert.AreEqual(expectedViewName, elevationViewName);
```

Comment:
need to add a check that the ElevationMarker's index that we specified is now occupied